### PR TITLE
[UI][Maxell] about page

### PR DESCRIPTION
**Pull request by maxellmilay**
What's in the PR?
 - entire about page
 - history, mission, and statement sections
 - fixed overlapping absolute images with header
 - fixed spacing issues

Demo Link:
https://www.loom.com/share/5392f58e9f594598b3bad191accb2f56

### CODE REVIEW

File: components/FullSection.tsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+export default function FullSection(props: PropsInterface) {
+    const { children, className } = props;
+
+    return (
+        <Section className={`h-screen ${className}`}>
+            {children}
+        </ Section >
+    )
+}
```
Comment (ezerssss) [inline components/FullSection.tsx]: why paman need ni nga component? what if i add nalang diay na nga class names sa `Section` nga component? or lets say prop nalang sa `Section` nga component nga lets say `isFull` ba ron, kay sahay oks ra nga dili kaayo abstracted ang mga stuff, as much as possible simpol lang ang buhay

File: components/Header.tsx [hunk 2]
```diff
@@ -59,10 +59,10 @@ function Header() {
   );
 
   return (
-    <>
+    <div className="fixed z-30">
```
Comment (ezerssss) [inline components/Header.tsx]: why gi addan div?

File: components/WhiteLine.tsx [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+interface PropsInterface {
+    className?: string
+}
+
+export default function WhiteLine(props: PropsInterface) {
+    const { className } = props;
+
+    return (
+        <div className={`h-2 bg-white ${className}`} />
```
Comment (ezerssss) [inline components/WhiteLine.tsx]: hmmm not sure if we really need nga usa ka component for a white line, can't we just put this in the page directly? unsa man and pros ani? like if gamiton nato on other components kay one place rata mag change? pero how many times do we really use this diay? basin too much abstraction raba nya mu pile up raba na ang file size if daghan ta components, hmmm

Comment (ezerssss) [reply components/WhiteLine.tsx]: also dili ba `hr` imo gina find? hehehehehehe

File: components/about/MissionDetail.tsx [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+}
+
+export default function MissionDetail(props: PropsInterface) {
+    const { className, title, info } = props;
+
+    return (
+        <div className={`flex flex-col md:flex-row justify-center items-center md:mr-[5vw] ${className}`}>
+            <h3 className='flex justify-center md:justify-end md:mr-10 font-bold w-[70vw] md:w-[35vw] text-4xl mb-3'>{title}</h3>
+            <p className='text-center md:text-left w-[70vw] md:w-[30vw] text-lg'>{info}</p>
+        </div>
```
Comment (ezerssss) [inline components/about/MissionDetail.tsx:16]: mao ni ang nice abstraction very nice

Comment (ezerssss) [reply components/about/MissionDetail.tsx:16]: ig implement nato sa cms kay ato ra i map with an array of data very very nice i love u max

**Review by ezerssss — CHANGES_REQUESTED** [review]

File: components/Header.tsx (hunk 2 above)
Comment (maxellmilay) [reply components/Header.tsx]: naa koy gi absolute nga pic, and I mu overlap siya ig wala gi set ang z layer sa outer div. Here's the pic kung walay outer z layer div:
image

File: components/FullSection.tsx (hunk 1 above)
Comment (maxellmilay) [reply components/FullSection.tsx]: resolved

**Review by ezerssss — APPROVED** [review]

**Pull request merged**